docs/static/azure-module.asciidoc

```diff
@@ -50,128 +52,98 @@ These {kib} dashboards are available and ready for you to use. You can use them
 models. 
 
-[[command-line-sample]]
-===== Command line configuration sample
-
-You can use the command line to set up the basic configuration for a single
-Event Hub. This command starts the Azure module with command line arguments,
-bypassing any settings in `logstash.yml`.
-
-["source","shell",subs="attributes"]
------
-bin/logstash --modules azure -M "azure.var.elasticsearch.host=es.mycloud.com" -M "azure.var.input.azure_event_hubs.threads=8" -M "azure.var.input.azure_event_hubs.consumer_group=logstash" -M "azure.var.input.azure_event_hubs.decorate_events=true" -M "azure.var.input.azure_event_hubs.event_hub_connections=Endpoint=sb://example1...EntityPath=insights-logs-errors" -M "azure.var.input.azure_event_hubs.storage_connection=DefaultEndpointsProtocol=https;AccountName=example...."
------
-
-The command line configuration requires the `azure.var.input.azure_event_hubs.` prefix before a configuration option.
-Notice the notation for the `threads` option. 
-
 [[basic-config-sample]]
 ===== Basic configuration sample
 
 The configuration in the `logstash.yml` file is shared between Event Hubs.
+Basic configuration is recommended for most use cases
 
 ["source","shell",subs="attributes"]
 -----
 modules:
   - name: azure
-    var.elasticsearch.hosts: "localhost:9200"
-    var.kibana.host: "localhost:5601"
-    var.input.azure_event_hubs.threads: 8
-    var.input.azure_event_hubs.decorate_events: true
-    var.input.azure_event_hubs.consumer_group: "logstash"
-    var.input.azure_event_hubs.storage_connection: "DefaultEndpointsProtocol=https;AccountName=example...."
+    var.elasticsearch.hosts: localhost:9200
+    var.kibana.host: localhost:5601
+    var.input.azure_event_hubs.consumer_group: "logstash" <1>
+    var.input.azure_event_hubs.storage_connection: "DefaultEndpointsProtocol=https;AccountName=instance1..." <2>
+    var.input.azure_event_hubs.threads: 9 <3>
     var.input.azure_event_hubs.event_hub_connections:
-      - "Endpoint=sb://example1...EntityPath=insights-logs-errors"
-      - "Endpoint=sb://example2...EntityPath=insights-metrics-pt1m"
+      - "Endpoint=sb://...EntityPath=insights-operational-logs" <4>
+      - "Endpoint=sb://...EntityPath=insights-metrics-pt1m" <5>
+      - "Endpoint=sb://...EntityPath=insights-logs-blocks"
+      - "Endpoint=sb://...EntityPath=insights-logs-databasewaitstatistics"
+      - "Endpoint=sb://...EntityPath=insights-logs-errors"
+      - "Endpoint=sb://...EntityPath=insights-logs-querystoreruntimestatistics"
+      - "Endpoint=sb://...EntityPath=insights-logs-querystorewaitstatistics"
+      - "Endpoint=sb://...EntityPath=insights-logs-timeouts"
 -----
 
+<1> The `consumer_group` (optional) is highly recommended. See <<azure_best_practices>>.
+<2> The `storage_connection` (optional) sets the Azure Blog Storage connection for tracking processing state for Event Hubs when scaling out a deployment with multiple Logstash instances. See "Scaling Event Hubs consumption" for additional details.
```

Comment:
Blog -> Blob

Reply:
@karenzone ^^

Reply:
Dang autocorrect from GDocs.   I've changed both instances in asciidoc source.